p4-16/spec/P4-16-spec.mdk

```diff
@@ -2539,34 +2539,54 @@ Operations that manipulate tuple types are described in Section
 
 The type `tuple<>` is a tuple type with no components.
 
+### List types { #sec-list-types }
+
+A list holds zero or more values, where every element must have the
+same type.  The type of a list where all elements have type `T` is
+written as
+
+~ Begin P4Example
+list<T>
+~ End P4Example
+
+~ Begin P4Grammar
+p4listType
```

Comment:
Comment from 2022-Dec-05 LDWG meeting: Add p4listType to where it belongs as the right-hand side of a grammar rule.

Reply:
I added p4listType to the definition of the nonterminal `typeRef` elsewhere in the spec where `typeRef` was defined.  That is the only occurrence of `p4listType` in the grammar on the right-hand side of a rule.